migrations/versions/dfeba9ccfc46_create_trading_as_field_in_business_.py

```diff
@@ -0,0 +1,35 @@
+"""create trading_as field in business_attributes
+
+Revision ID: dfeba9ccfc46
+Revises: 2798b5d5566a
+Create Date: 2018-04-12 12:11:59.813999
+
+"""
+from alembic import op
+import json
+
+
+# revision identifiers, used by Alembic.
+revision = 'dfeba9ccfc46'
+down_revision = '2798b5d5566a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+
+    sql_query = "SELECT * FROM partysvc.business_attributes"
+    result = conn.execute(sql_query)
+    for row in result:
+        attributes = json.loads(json.dumps(row['attributes']))
+        trading_as = '{tradstyle1} {tradstyle2} {tradstyle3}'.format(**attributes)
+        attributes['trading_as'] = ' '.join(trading_as.split())
```

Comment:
why are you doing a split and then join again here?  attributes['trading_as'] = trading_as?

Reply:
gets rid of any spaces if any of them are empty.

Reply:
Better solution to this is `trading_as.strip()` removed leading and trailing spaces

Reply:
it's the double space inbetween tradstyle1 and tradstyle3 if tradstyle2 is empty

Reply:
with strip if tradstyle2 is blank it will end up with "tradstyle1 tradstyle2" rather than "tradstyle1tradstyle2"